api/package.json

```diff
@@ -66,6 +63,9 @@
         "supertest": "^6.2.3"
     },
     "devDependencies": {
+        "@aws-sdk/client-s3": "^3.360.0",
```

Comment:
<img width="910" alt="image" src="https://github.com/JiscSD/octopus/assets/48569671/2e60db41-33a4-4a40-8935-1b4c5aa7cd01">

all these packages must be excluded from the bundle with 'forceExclude' in serverless.yml
You can take quick a look on my PR to see an example how i excluded puppeteer https://github.com/JiscSD/octopus/pull/430/files#:~:text=includeModules%3A-,forceExclude,-%3A%20puppeteer